package.json

```diff
@@ -7,6 +7,8 @@
     "start": "webpack-dev-server --open",
     "build": "webpack",
     "test": "jest",
+    "clean": "rm -rf node_modules",
+    "reinstall": "npm run clean && npm install",
```

Comment:
should also probably add `npm cache clean` here so we dont have any persisting modules from cache